lib/Web2Print/Processor/HeadlessChrome.php

```diff
@@ -113,8 +113,8 @@ public function getPdfFromString($html, $params = [], $returnFilePath = false)
         $converter = new Converter($input, $output);
         if ($this->nodePath) {
             $converter->setNodePath($this->nodePath);
-        } else {
-            $converter->setNodePath(Console::getExecutable('node'));
+        } elseif ($nodePath = Console::getExecutable('node')) {
```

Comment:
Does this really change anything? The `$nodePath` can still be `true` and therefore will fail?

Reply:
`Console::getExecutable()` returns `string|false`